### PR TITLE
Fix ncss = 0

### PR DESCRIPTION
**Pull request by lyriccoder**
https://github.com/yegor256/aibolit/issues/461

### CODE REVIEW

File: aibolit/__main__.py [hunk 1]
```diff
@@ -242,11 +242,18 @@ def run_recommend_for_file(file: str, args):
...
+        error_string = 'Empty java file; ncss = 0'
+    else:
+        results_list = inference(input_params, code_lines_dict, args)
+
     if error_string:
-        ncss = 0
+        ncss = 0.000000000000001
     else:
-        ncss = input_params.get('M4', 0)
+        ncss = input_params.get('M4', 0.000000000000001)
```
Comment (acheshkov) [inline aibolit/__main__.py:256]: suggest to remove

Comment (acheshkov) [inline aibolit/__main__.py:256]: suggest to remove

Comment (acheshkov) [inline aibolit/__main__.py:256]: suggested to remove

**Comment by acheshkov** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/aibolit#464` is not assigned, can't get performer

**Pull request merged**